### DISCUSSION TITLE
MTG/packs/snc-01

### DISCUSSION THREAD

**Discussion by giscus** [Nerban/MTG#6] [BoosterComments]
# MTG/packs/snc-01

https://nerban.github.io/MTG/packs/snc-01

**Comment by Nerban** [comment]
Hey, I hope you enjoyed this booster pack!

> -  What do you think is the theme of the (un)common cards in this set booster?
>
>  - What's your favorite?

Greetings

**Reply by PittiBuh** [reply]
I think the Theme of the uncommon cards ist backstab, or treason, even the Wildcard gives a vibe of it. In a Racers ring you can have much of corruption going.
The Commander does not fit that much in my Themes, but i think in downtown u have much going on with bad stuff, so maybe kinda going with it.
Think the Interceptor fits well in the idea of corruption headhunting and backstab.

My favorite card is Call in a Professional, i like the Background of that drawing and its effect feels great just from reading.

Have a nice day!

**Reply by Nerban** [reply]
Yes, I also thought about dirty fighting, either in the artwork or the card effect itself.

I think only the commons are put together under a common theme, which can exploit a mechanic, a story or any kind of link.

My favorite Card of this set is actually the Quick Draw Dagger.
The Art, the Name and the effect just fit very good together in my opinion.
